src/_util.js

```diff
@@ -451,7 +451,17 @@ function mergeObjects(obj1, obj2) {
   return obj3
 }
 
+var crossGlobal =
```

Comment:
Could you maybe add some docs here explaining the motivation for this? Abdicating from using global `window` variable might be counter intuitive without further context.

Reply:
almost all platforms supports globalThis, which is kinda built-in polyfill for global context to cover cross platforms. but we can't use it as IE11 doesn't support it https://caniuse.com/?search=globalThis

window - browser global context
global - nodejs global context
self - service worker global context

so crossGlobal will contain one of them depends on availability

Reply:
That's good information, would you mind adding that as a comment to the code?